### DISCUSSION TITLE
Hide/Show multiple mesh object in the single glb

### DISCUSSION THREAD

**Discussion by aiswaryapramod** [google/model-viewer#4739] [Q&A] [answered]
When using Model-Viewer, I want to hide or show a specific material from a model. Could you please provide an example link?

**Comment by danilokrulj** [comment, marked as answer]
Did you have a look at the examples?

https://modelviewer.dev/examples/scenegraph/#hideShowMeshVariantsExample

**Reply by aiswaryapramod** [reply]
The provided link is not working for me, could you please provide a GitHub repository that contains the feature that i am looking for?

**Reply by elalish** [reply]
I believe they gave you the ideal link. What about that example isn't working for you?

**Reply by aiswaryapramod** [reply]
Now its working. Thank you.
